modules/app/app-contentstudio/src/main/js/app/wizard/page/contextwindow/inspect/page/PageInspectionPanel.ts

```diff
@@ -66,9 +78,30 @@ export class PageInspectionPanel extends BaseInspectionPanel {
 
         this.pageControllerSelector = new PageControllerSelector(this.liveEditModel);
         this.pageControllerForm = new PageControllerForm(this.pageControllerSelector);
+        this.pageControllerForm.onShown(event => this.updateSaveAsTemplateVisibility());
         this.pageControllerForm.hide();
         this.appendChild(this.pageControllerForm);
 
+        this.currentUserHasCreateRights = null;
+        this.saveAsTemplateButton = new Button(i18n('action.saveAsTemplate'));
+        this.saveAsTemplateButton.addClass('blue large save-as-template');
```

Comment:
See comment above - this should be an ActionButton linked to SaveAsTemplate action.